sagemaker-debugger/tensorflow2/tensorflow2_keras_to_estimator/tf2-keras-model-to-estimator.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>Once the rule job starts, this will return RuleEvaluationJabArn values. We can see the logs for the rule job in CloudWatch. To do that, we will use the following utlity functions to get the CloudWatch link to the rule job logs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>done in latest</p>